enterprise-suite/scripts/lbc.py

```diff
@@ -284,6 +284,36 @@ def check_resource_list(cmd, expected, fail_msg):
         return all_found
     return False
 
+# This is the behavior we'll have to implement to be in line with the thinking of
+# https://github.com/lightbend/es-backend/issues/572
+
+# // This first case would be typical for a dev/demo.  Chances are we're okay losing the data.
+# If    we're changing from usePersistentVolumes=false to usePersistentVolumes=true
+#       (which means going from emptyDir volume to PV)
+#    or
+#       we're about to "helm delete" and usePersistentVolumes was false
+# then
+#    warn user that they will lose their Console data
+#    (Not sure how useful this is really.  Don't think they can (easily) grab the data.)
+##
+# // This case is the nasty one.  Chance of losing real data here.
+# If    we're changing from usePersistentVolumes=true to usePersistentVolumes=false
+#       (which means going from PV to emptyDir volume)
+#    or
+#       we're about to "helm delete" and usePersistentVolumes was true
+# then
+#    if PVC reclaim policy is not Retain
+#       warn user that they will lose their Console data
+#    else
+#       warn that their data will not be deleted but they'll have to do some manual steps
+#       - if they want to reuse it, or
+#       - if they actually want to delete it
+#
```

Comment:
I think this pseudo code matches the steps I listed already, right? https://github.com/lightbend/es-backend/issues/572#issuecomment-460725610

Reply:
I think to so too.  The only possible caveat is that it is careful to consider the current state as well, not just the desired state.